### PR TITLE
docs: Clean-up examples. Fixes #4124

### LINKED ISSUE #4124: Documentation: Examples edit link is wrong
Hello lovely creators of this great tool!

Just getting started, but already half in love.

I wanted to edit the examples page, but the link seems to be wrong (returns 404):
https://github.com/argoproj/argo/edit/master/docs/examples/README.md

It looks like `-w` is removed and the second line gets all steps, is that correct?

```
argo logs -w hello-world-xxx    # get logs from all steps in a workflow
argo logs hello-world-xxx-yyy   # get logs from a specific step in a workflow
```

### PR DESCRIPTION
**Pull request by secretlifeof**
Checklist:

* [b] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.
* [c] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [x] I've signed the CLA.
* [no] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My builds are green. Try syncing with master if they are not. 
* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo/blob/master/USERS.md).

Fixes #4124

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -184,24 +183,24 @@ spec:
...
-        value: "{{workflow.parameters.log-level}}"
-      command: [runA]
-  - name: B
-    container:
-      image: containerB
-      env:
-      - name: LOG_LEVEL
-        value: "{{workflow.parameters.log-level}}"
-      command: [runB]
+    - name: A
```
Comment (alexec) [inline examples/README.md:190]: please can you revert these whitespace changes?

**Review by alexec — CHANGES_REQUESTED** [review]
Please revert whitespace changes.

**Comment by alexec** [comment]
Bump! Close or continue?

**Comment by secretlifeof** [comment]
Sorry. For being so extremely late. I have copied the formatting from the raw markdown file. What do you think about creating a **.vsc** folder with format settings so that this will not happen?

**Review by alexec — APPROVED** [review]
I don't feel strongly about white-space.

**Comment by secretlifeof** [comment]
Next time it will not happen

**Pull request merged**